css/myparcel.css

```diff
@@ -191,6 +192,9 @@
 #mypa-list-postnl-pickup option{
 	padding: 5px 0px 5px 0px;
 }
+.mypa-boldPrice{
```

Comment:
Moet zijn: `.mypa-bold-price` of `.mypa-bold_price`

Reply:
Done